consts/models.js

```diff
@@ -12,7 +12,8 @@ const refs = {
   NOTIFICATIONS: 'Notifications',
   COMMENT: 'Comment',
   LESSON: 'Lesson',
-  COURSE:'Course'
+  COURSE: 'Course',
+  QUIZ: 'QUIZ'

```

Comment:
I guess, only first character should be in upper-case

Reply:
fixed